femm-unofficial.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
```

Comment:
since you supply this file, it should have the proper name from start.

Reply:
Can you help me with that? Not sure if I follow.

Reply:
From the documentation you acknowledged having read: https://github.com/flathub/flathub/wiki/App-Maintenance#acceptable-but-should-be-submitted-upstream

(I don't expect it to be submitted upstream for obvious reaso, but the rest stands)

Reply:
Yes. I've read. However, this my first build and I'm prone to fail. I really appreciate your effort and help. 
You mean the file name right? I've missed that.